enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/RequestContext.java

```diff
@@ -53,6 +53,7 @@
     private Map<String, String> responseHeaders;
     private Map<String, String> metadataMap = new HashMap<>();
     private String requestPathTemplate;
+    private String traceId;
```

Comment:
We are just keeping this in the request context for future usages and not needed for login the traceId ryt?